### PR TITLE
feat: use headers attribute if available

### PR DESCRIPTION
**Pull request by thesayyn**
This renders the initial setup step for people using `>=Bazel 7.1`. 

People who has done the initial setup may do the opposite of what was instructed in the https://github.com/chainguard-dev/rules_apko/blob/main/docs/initial-setup.md

### CODE REVIEW

File: apko/private/bazelversion.bzl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        # An empty string is treated as a "dev version", which is greater than anything.
+        v = "999999.999999.999999"
+    release, prerelease = _partition(v.partition(" ")[0])
+    segments = release.split(".")
+    if len(segments) != 3:
+        fail("invalid Bazel version '{}': got {} dot-separated segments, want 3".format(v, len(segments)))
+    return [_safe_int(s, v) for s in segments], not prerelease, prerelease
+
+def bazel_version_gte(version):
+    return parse_version(native.bazel_version) >= parse_version(version)
```
Comment (alexeagle) [inline apko/private/bazelversion.bzl:45]: You can get this from https://github.com/bazelbuild/bazel-skylib/blob/main/docs/versions_doc.md - don't need bazel_features I believe

Comment (thesayyn) [reply apko/private/bazelversion.bzl:45]: oh nice, using that now

**Review by imjasonh — APPROVED** [review]

**Comment by hunshcn** [comment]
Is this blocked by something?

**Comment by hunshcn** [comment]
hi @alexeagle

**Comment by hunshcn** [comment]
@thesayyn Can I know why the merger is being blocked?

**Pull request merged**